src/pybind/rados/rados.pyx

```diff
@@ -249,7 +249,7 @@ cdef extern from "rados/librados.h" nogil:
     void rados_ioctx_snap_set_read(rados_ioctx_t io, rados_snap_t snap)
     int rados_ioctx_snap_list(rados_ioctx_t io, rados_snap_t * snaps, int maxlen)
     int rados_ioctx_snap_get_stamp(rados_ioctx_t io, rados_snap_t id, time_t * t)
-    uint64_t rados_ioctx_get_id(rados_ioctx_t io)
```

Comment:
@jdurgin this is the change i mentioned the other day when discussing the API doc build.